### PR TITLE
aws-elasticache 1.12

### PR DESCRIPTION
**Pull request by bfontaine**

### CODE REVIEW

File: Library/Formula/aws-elasticache.rb [hunk 1]
```diff
@@ -1,12 +1,10 @@
-require 'formula'
-
 class AwsElasticache < AmazonWebServicesFormula
-  homepage 'http://aws.amazon.com/developertools/2310261897259567'
-  url 'https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-2013-06-15-1.9.000.zip'
-  version '1.9.000'
-  sha1 'c8962cc2ea22f88883b38cda2674d1befbed1b71'
+  homepage "http://aws.amazon.com/developertools/2310261897259567"
+  url "https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-latest.zip"
```
Comment (tdsmith) [inline Library/Formula/aws-elasticache.rb]: We need a versioned URL for this.

Comment (bfontaine) [inline Library/Formula/aws-elasticache.rb]: Yes, can’t find one I asked an AWS Support account on Twitter about this.

**Comment by adamv** [comment]
Note that we have other AWS tools in master that don't have versioned downloads; they don't update very often and I wouldn't be too big of a stickler on this.

Alternate take: move these to homebrew-aws.

**Comment by vszakats** [comment]
`aws.amazon.com` domain supports proper HTTPS, so I'd suggest to update `homepage` to use it:
https://aws.amazon.com/developertools/2310261897259567

**Comment by bfontaine** [comment]
@vszakats Right, just edited the formula :+1:

**Comment by vszakats** [comment]
@bfontaine Thank you.

File: Library/Formula/aws-elasticache.rb [hunk 2]
```diff
@@ -1,14 +1,18 @@
-require 'formula'
-
 class AwsElasticache < AmazonWebServicesFormula
-  homepage 'http://aws.amazon.com/developertools/2310261897259567'
-  url 'https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-2013-06-15-1.9.000.zip'
-  version '1.9.000'
-  sha1 'c8962cc2ea22f88883b38cda2674d1befbed1b71'
+  homepage "https://aws.amazon.com/developertools/2310261897259567"
+  url "https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-latest.zip"
```
Comment (mikemcquaid) [inline Library/Formula/aws-elasticache.rb]: We don't like using `-latest` links as they break on new versions. Can you try and find a versioned one instead? Thanks!

Comment (bfontaine) [inline Library/Formula/aws-elasticache.rb]: @mikemcquaid see [my earlier comment](https://github.com/Homebrew/homebrew/pull/36012#discussion_r23147667)

**Comment by vszakats** [comment]
This is the filename of the current latest version:
```
https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-2014-09-30-1.12.000.zip
```

Found out using:
```bash
curl -f -O https://s3.amazonaws.com/elasticache-downloads/AmazonElastiCacheCli-2014-[09-10]-[01-31]-1.12.000.zip
```

**Comment by bfontaine** [comment]
Awesome ! I tried a couple different URLs too but didn’t find the good one.

**Pull request closed without merging**